package.json

```diff
@@ -9,7 +9,8 @@
     "skyux": "skyux",
     "start": "skyux serve --hmr",
     "test": "skyux test --coverage library --logFormat none",
-    "test:visual": "skyux e2e --logFormat none"
+    "test:visual": "skyux e2e --logFormat none",
+    "prepare": "npm i --no-save --force @skyux/core"
```

Comment:
This line is needed because NPM will not install a package that shares the same `name` as the project. We need `@skyux/core`, however, because it is a peer dependency for many packages.